### PR TITLE
ENHANCEMENT Backport versioned querystring fix

### PR DESCRIPTION
**Pull request by tractorcow**
Parent issue https://github.com/silverstripe/silverstripe-versioned/issues/145

### CODE REVIEW

**Comment by tractorcow** [comment]
Requires https://github.com/silverstripe/silverstripe-framework/pull/8026

File: tests/controller/ContentControllerTest.php [hunk 1]
```diff
@@ -7,16 +7,22 @@ class ContentControllerTest extends FunctionalTest {
 
 	protected static $fixture_file = 'ContentControllerTest.yml';
 
-	protected static $use_draft_site = true;
-
 	protected static $disable_themes = true;
 
+	public function setUp()
+	{
+		parent::setUp();
```
Comment (unclecheese) [inline tests/controller/ContentControllerTest.php]: Let's keep block formatting consistent with the rest of the class.

File: tests/model/RedirectorPageTest.php [hunk 2]
```diff
@@ -2,47 +2,59 @@
...
 class RedirectorPageTest extends FunctionalTest {
 	protected static $fixture_file = 'RedirectorPageTest.yml';
-	protected static $use_draft_site = true;
+
 	protected $autoFollowRedirection = false;
-	
+
+	public function setUp()
+	{
+		parent::setUp();
```
Comment (unclecheese) [inline tests/model/RedirectorPageTest.php]: Same here. Move to same line.

Comment (tractorcow) [reply tests/model/RedirectorPageTest.php]: Ah right, reverse linting. ;)

**Comment by unclecheese** [comment]
Have confirmed that tests are only failing because they depend on the [framework PR](https://github.com/silverstripe/silverstripe-framework/pull/8026). We should merge this ahead of green, as the PRs are interdependent.

**Pull request merged**